aspnetcore/performance/caching/response/sample/ResponseCacheSample.xproj

```diff
@@ -4,20 +4,20 @@
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
-  <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" Condition="'$(VSToolsPath)' != ''" />
+  <Import Project="$(VSToolsPath)\DotNet\Microsoft.DotNet.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
-    <ProjectGuid>3ef40110-98b1-4b6e-8e06-ad21cd8da869</ProjectGuid>
+    <ProjectGuid>ee294ce9-2562-4ef2-9b2a-28b1e823b904</ProjectGuid>
     <RootNamespace>ResponseCacheSample</RootNamespace>
-    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
-    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
+    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">.\obj</BaseIntermediateOutputPath>
+    <OutputPath Condition="'$(OutputPath)'=='' ">.\bin\</OutputPath>
+    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
```

Comment:
I figured it out, the `Vary: Accept-Encoding` header was being appended by the IIS Static/Dynamic Compression. If you run the app without IIS or disable the compression modules, you'll see the expected behavior. This is a little tricky so I'm not sure if we want to mention it in the docs. There's also not a good way to modify the sample since the IIS apphost.config is built dynamically and you cannot remove IIS Express from the launchsettings.json.